src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx

```diff
@@ -76,25 +80,22 @@ export const ThankYou: React.FC = () => {
 
       <Flex
         pb={2}
-        width={["100%", "60%"]}
+        width={["100%", "80%", "50%"]}
         flexDirection={["column", "row"]}
-        alignItems={["stretch", "center"]}
+        alignItems={"center"}
       >
-        <RouterLink to="/sell/submission">
-          <Button
-            mr={[0, 150]}
-            width="100%"
-            data-test-id="submit-another-work"
-            size="large"
-            variant="primaryBlack"
-            onClick={trackSubmitAnotherWorkClick}
-          >
-            Submit Another Work
-          </Button>
-        </RouterLink>
+        <Button
+          width={["100%", "50%"]}
+          data-test-id="submit-another-work"
+          size="large"
+          variant="primaryBlack"
+          onClick={submitAnotherWork}
+        >
+          Submit Another Work
+        </Button>
         <Spacer x={[0, 2]} y={[2, 0]} />
         <Button
-          width="100%"
+          width={["100%", "50%"]}
```

Comment:
The buttons' size is different. Is it intentional?
`width={["100%", "80%", "50%"]} ` / `width={["100%", "50%"]}`

Reply:
@dariakoko . Yes it is. 100 for mobile, 80 for pads, 50 for desktop.

Reply:
I mean "Submit Another Work" button has three size options and "View Artwork in My Collection" has just two

Reply:
Buttons are never really percentage widths other than `100%`, they typically should be sized to occupy column spans in our grid system.

Reply:
@dariakoko That is actually the container. The buttons are the children here. But I have refactored to use Columns.

Reply:
Thanks @dzucconi